cli/internal/core/engine.go

```diff
@@ -107,31 +98,54 @@ func (e *Engine) Execute(visitor Visitor, opts EngineExecutionOptions) []error {
 }
 
 func (e *Engine) getTaskDefinition(taskName string, taskID string) (*Task, error) {
-	if task, ok := e.Tasks[taskID]; ok {
-		return task, nil
+	pipeline, err := e.getPipelineFromWorkspace(util.RootPkgName)
```

Comment:
Hardcoding the lookup from root pipeline, but we'll expand it to starting in the actual packageTask's workspace.